docs/guides/slp-software-setup.md

```diff
@@ -0,0 +1,367 @@
+```
+
+Make sure everything is installed without errors.
+
+
+## SLP wallet
+Before moving on with the scripting, you’ll need some SLP addresses of your own to work with. If you already have some, feel free to skip these steps.
+
+First off, let’s get you an SLP compatible wallet. This will be the place where your freshly minted SLP tokens will reside, and also where you’ll have to send the required BCH to do so. After installing the software, we will make sure to keep the SLP address and it’s WIF.
+
+If you'd rather try things on testnet, you'll find a small comment after each section indicating the changes needed. To get testnet coins you can find several services, like the [bitcoin.com Testnet BCH faucet](https://developer.bitcoin.com/faucets/bch/).
+
+### Option 1: Electron Cash SLP
+
+#### GUI
+
+[Check out our Electron Cash SLP edition walkthrough](/guides/slp-software-setup) **(Coming soon)**
+
+#### CLI
+
+**SLP wallet creation**
+
+```bash
+> Electron-Cash-SLP-3.6.4.exe create_slp
+
+Password (hit return if you do not wish to encrypt your wallet):
+Your wallet type is: slp_standard
+Your wallet generation seed is:
+"dinner valley cattle brush moment critic diagram party crack frost tilt shock"
+Please keep it in a safe place; if you lose it, you will not be able to restore your wallet.
+Wallet saved in 'c:\users\<user>\appdata\roaming\electroncash\wallets\default_wallet'
+```
+
+**List addresses in wallet**
+
+```bash
+> Electron-Cash-SLP-3.6.4.exe listaddresses
+
+[
+    "qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh",
+    "qr70zmq9l7fyrnu4f9nz4fzp5a80zrfppc3s3w90lf",
+    "qqekzmw5tgxaqfps0f63wgjnx38q8jfv8cjlf8vzy5",
+    "qz6rgncgck5qa6ez8ej7t2yqc4yt9zmanyqxs5uxxf",
+    "qrlxewwqqsen2ude8alh6p50xd3tekdaqv28h3qsh7",
+    "qru0k6ws0dx44699xkvw7gmxpfavj9ll4sqkqm6vge",
+    ...
+]
+```
+
+
+
+**Get the private key from an address**
+
+```bash
+> Electron-Cash-SLP-3.6.4.exe getprivatekeys qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh
+
+WARNING: ALL your private keys are secret.
+Exposing a single private key can compromise your entire wallet!
+In particular, DO NOT use 'redeem private key' services proposed by third parties.
+L1aeBNDeQ43PtFN2Jc9tbeqhNiqBgDHbxMctefHftHw4WtSo621e
+```
+
+As a last step, in order to send the needed BCH to the funding address, we need to make sure to use the Cash address format. For that conversion we will use the following call
+
+```bash
+> Electron-Cash-SLP-3.6.4.exe addressconvert qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh
+
+{
+    "cashaddr": "bitcoincash:qzvjfs5tuw56est4mwtzw3fegucgdwr00vrgaayqsf",
+    "legacy": "1ExkRUc6r128pvMkqwLGiLgfnmmdyYL6BA",
+    "simpleledger": "simpleledger:qzvjfs5tuw56est4mwtzw3fegucgdwr00v0nkx3qwh"
+}
+```
+
+After this, we will have generated the three necessary bits of information we need:
+
+**The WIF (private key)**    
+`L1aeBNDeQ43PtFN2Jc9tbeqhNiqBgDHbxMctefHftHw4WtSo621e`
+
+**The recieving SLP address, make sure to add the prefix ‘simpleledger:’**    
```

Comment:
reciving spelling. also "to add" is kinda confusing since the checksum is different (this was my misunderstanding when reading, but probably could happen as well), maybe word differently or further explain?

Reply:
Uhh I think at first I was under the impression that it was the same address with different prefix at first; then saw the addressconverter and found out it is not the case, so the comment just makes no sense and I forgot to remove it.
Fixed.